global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "msbuild-sdks": {
+        "Microsoft.Build.CentralPackageVersions": "2.0.52"
+    }
+}
```

Comment:
Add terminating newline or a .editorconfig

Reply:
https://github.com/messerli-informatik-ag/linq-to-rest/pull/35/commits/a23cbc48eaf580a955ce0cc5c7f9b6b727e92a15